autoload/go/term.vim

```diff
@@ -74,8 +74,8 @@ function! go#term#newmode(bang, cmd, errorformat, mode) abort
           \ 'exit_cb' : function('s:exit_cb', [], state),
         \ }
 
-    if l:mode =~ "vertical" || l:mode =~ "vsplit" || l:mode =~ "vnew"
-      let l:term["vertical"] = 1
+    if !(l:mode =~ "split") || !(l:mode =~ "new")
```

Comment:
@bhcleek 

Does this accomplish the goal? If we do not see l:mode set to the normal split or new modifiers, we then populate l:term["vertical"] with which ever value the user placed in their config. Assuming "split" an "new" are not modifiers... but maybe I'm wrong.

Reply:
The option could be `belowright`. I think you want to keep the original condition and this change's assignment to `l:term`.

Reply:
ahh I see now, I haven't used these modifiers before. 
So the correct thing to do would be 

```
if l:mode =~ "vertical" || l:mode =~ "vsplit" || l:mode =~ "vnew"
      let l:term["vertical"] = l:mode
endif
```

and leave it at that?

Reply:
Yes